data/human/human missions.txt

```diff
@@ -7411,3 +7411,1825 @@ mission "Adelita 5"
+			`	Except, of course, the lack of one familiar face. A doomed and disgraced ex-governor who had been sent here, and last you heard, had died here.`
+			`	With that heavy thought on your mind you notice a sign which promises to lead you to the local graveyard.`
+			choice
+				`	(Head to the graveyard.)`
+					goto graveyard
+				`	(Head back to your ship.)`
+
+			`	You see no point in wasting your time at this place's graveyard, so you head back to your ship.`
+				decline
+			
+			label graveyard
+			`	Oblivion's evening sky is lit up by unnumbered rigs and natural gas burners that dot the region. These red fires turn the normally dark blue atmosphere a shade of smoldering purple.`
+			`	It's beneath that purple hue you see a countless expanse of small flat gravestones uniformly placed in Oblivion's sterile dirt. Despite the number of headstones there is no one else here, and only a few graves show any evidence of being visited.`
+			`	It seems as though those buried here are mostly forgotten.`
+			choice
+				`	(Search for Timothy Radrickson's grave.)`
+					goto search
+				`	(Leave.)`
+
+			`	You've seen all you wanted to see of this place, and leave this unloved graveyard to its forlorn dead.`
+				decline
+
+			label search
+			`	The index is easy to find, and has all the hallmarks of having never been used. The graveyard is set up in a carefully regimented grid system and despite the large number of graves, it is surprisingly easy finding Timothy's.`
+			`	You walk across the forgotten memorial to find a small, flat unremarkable stone, hemmed in amongst other unremarkable stones, pressed into the dead ground. The text laser-chiseled onto it reads, "Timothy Radrickson. Prisoner - 645631-4C"`
+			`	That's all. Nothing about his meteoric rise, about being the Governor of Rand, or even how or when his death occurred here. Just another dead prisoner.`
+			choice
+				`	(Stay a while and contemplate.)`
+				`	(Leave.)`
+					goto end
+
+			`	The slow moving evening clouds have cast the entire world in a strange purple twilight. Timothy's grave sits before you, nestled in the poison ground, and you alone in the vast empty graveyard.`
+			`	Your thoughts drift to the man buried beneath your feet.`
+			label stayawhile
+			choice
+				`	(Think about Timothy's past.)`
+					goto past
+					to display
+						not "past"
+				`	(Think about Timothy's rise.)`
+					goto rise
+					to display
+						not "rise"
+				`	(Think about Timothy's demise.)`
+					goto demise
+					to display
+						not "demise"
+				`	(Leave the graveyard.)`
+					goto end
+
+			label past
+			action
+				set "past"
+			`	You never knew what Timothy had been fleeing when he first stowed away on your ship. Who he had been, or what he had done that lead him to such dire straits.`
```

Comment:
All of these descriptions feel better suited for a logbook entry than as part of a conversation.

Reply:
I am okay with that. I've no idea how the logbook works though, or really understand it/used it on my play through.

Reply:
You can find basic information on how to create logs in a mission by looking at https://github.com/endless-sky/endless-sky/wiki/CreatingMissions. If you look in human missions.txt you can see examples of logs for people you meet during various missions. I'd adapt the stuff you've written here as logs throughout this mission chain, because it feels weird to go through this entire mission chain without Timothy getting mentioned in the logbook.
For example, I'd use lines 9297-8 as log entries for right after you meet Timothy and after he pays you back. If you need help with this I can try making a PR to your branch, but I can't guarantee I'll be able to get it done soon.